system/index.js

```diff
@@ -88,7 +88,9 @@ module.exports = async (message) => {
     throw error;
   }
 
-  return result;
+  if (typeof result !== 'undefined') {
+    return result
```

Comment:
nit - do we have a style rule about semicolons?

Reply:
We don't that I know of, but I vote for using prettier and w/e is default on it.

Reply:
it's @oclif/tslint the both evergreen-cli and this evergreen build plugin uses, it disallows semicolon at the end

    "semicolon": [true, "never"],